nix_update/version/github.py

```diff
@@ -26,12 +26,15 @@ def version_from_entry(entry: Element) -> Version:
 def _dorequest(url: ParseResult, feed_url: str) -> str:
     request = urllib.request.Request(feed_url)
 
-    login, account, password = netrc.netrc().authenticators(url.netloc)
-    if login is not None:
-        info(f"using netrc file")
-        encoded = ('%s:%s' % (login, password)).encode('utf8')
-        creds = base64.b64encode(encoded).strip()
-        request.add_header("Authorization", "Basic %s" % creds.decode("utf8"))
+    try:
+        login, account, password = netrc.netrc().authenticators(url.netloc)
+        if login is not None:
+            info("using netrc file")
+            encoded = ('%s:%s' % (login, password)).encode('utf8')
+            creds = base64.b64encode(encoded).strip()
+            request.add_header("Authorization", "Basic %s" % creds.decode("utf8"))
+    except FileNotFoundError:
+        info("no netrc file found")
```

Comment:
I would lower this to debug

Reply:
there are just warn/info, so lets pass: https://github.com/h0nIg/nix-update/blob/patch-2/nix_update/utils.py#L23-L24